virtcontainers/utils/utils.go

```diff
@@ -203,3 +206,16 @@ func BuildSocketPath(elements ...string) (string, error) {
 
 	return result, nil
 }
+
+// SupportsVsocks returns true if vsocks are supported, otherwise false
+func SupportsVsocks() bool {
+	if _, err := os.Stat(VSockDevicePath); err != nil {
+		return false
+	}
+
+	if _, err := os.Stat(VHostVSockDevicePath); err != nil {
```

Comment:
Same here.

Reply:
what I can do is to return `bool` and `error` but that's odd